package.json

```diff
@@ -31,9 +31,18 @@
     "@typescript-eslint/parser": "^5.4.0",
     "eslint": "^8.2.0",
     "eslint-config-3box": "^0.4.0",
+    "ethers": "^5.5.1",
     "jest": "^27.3.1",
     "jest-resolver-enhanced": "^1.0.1",
     "prettier": "^2.4.1",
+    "regenerator-runtime": "^0.13.9",
     "typescript": "^4.4.4"
+  },
+  "dependencies": {
+    "@ipld/dag-cbor": "^6.0.14",
+    "@stablelib/sha256": "^1.0.1",
+    "apg-js": "^4.1.1",
+    "multiformats": "^9.5.1",
+    "multihashes": "^4.0.3"
```

Comment:
as hashes not included in `multiformats`

Reply:
might not need them with the blocks approach from multiformats, let me refactor and see if needed

Reply:
no need!